### PR TITLE
Missing search glass in search box of the documentation

### PR DESCRIPTION
**Pull request by albert-github**
Due to the change for the doxygen fix https://github.com/doxygen/doxygen/pull/7888 the file `mag_sel.png` is replaced by `mag_sel.svg`, as a result the search glass in the search box was not present anymore.
(Note this is only valid for versions 1.8.19 and newer).

### CODE REVIEW

**Comment by sloriot** [comment]
@maxGimeno do you know how to generate resources for a new doxygen version? If yes, I suggest we create a new one for 1.8.19.

**Comment by albert-github** [comment]
Better **NOT** to use 1.8.19 (Release date August 8, 2020) as it contains a number of serious bugs that have been fixed in 1.8.20 (Release date August 24, 2020)

**Comment by maxGimeno** [comment]
@sloriot not really. I guess it is all about copy-paste/adapt ?

**Comment by sloriot** [comment]
@lrineau  Shall I rebase the branch for 5.0.X?

File: Documentation/doc/resources/1.8.19/BaseDoxyfile.in [hunk 1]
```diff
@@ -0,0 +1,2613 @@
...
+# If the HTML_FORMULA_FORMAT option is set to svg, doxygen will use the pdf2svg
+# tool (see https://github.com/dawbarton/pdf2svg) or inkscape (see
+# https://inkscape.org) to generate formulas as SVG images instead of PNGs for
+# the HTML output. These images will generally look nicer at scaled resolutions.
+# Possible values are: png (the default) and svg (looks nicer but requires the
+# pdf2svg or inkscape tool).
+# The default value is: png.
+# This tag requires that the tag GENERATE_HTML is set to YES.
+
+HTML_FORMULA_FORMAT    = svg
```
Comment (sloriot) [inline Documentation/doc/resources/1.8.19/BaseDoxyfile.in:1603]: This one is a new option that I put to svg

File: Documentation/doc/resources/1.8.19/BaseDoxyfile.in [hunk 2]
```diff
@@ -0,0 +1,2613 @@
...
+# Note: If you choose svg you need to set HTML_FILE_EXTENSION to xhtml in order
+# to make the SVG files visible in IE 9+ (other browsers do not have this
+# requirement).
+# Possible values are: png, jpg, gif, svg, png:gd, png:gd:gd, png:cairo,
+# png:cairo:gd, png:cairo:cairo, png:cairo:gdiplus, png:gdiplus and
+# png:gdiplus:gdiplus.
+# The default value is: png.
+# This tag requires that the tag HAVE_DOT is set to YES.
+
+DOT_IMAGE_FORMAT       = png
```
Comment (sloriot) [inline Documentation/doc/resources/1.8.19/BaseDoxyfile.in:2498]: Shall we move to svg too?

File: Documentation/doc/CMakeLists.txt [hunk 3]
```diff
@@ -220,10 +220,15 @@ set(CGAL_DOC_DXY_DIR "${CMAKE_BINARY_DIR}/doc_dxy")
 file(MAKE_DIRECTORY "${CGAL_DOC_DXY_DIR}")
 
 #Setting the resource directory depending on the version of doxygen
-set(CGAL_DOC_RESOURCE_DIR_DEFAULT "${CMAKE_CURRENT_LIST_DIR}/resources/1.8.14")
+set(CGAL_DOC_RESOURCE_DIR_DEFAULT "${CMAKE_CURRENT_LIST_DIR}/resources/1.8.19")
```
Comment (albert-github) [inline Documentation/doc/CMakeLists.txt:223]: Don't use 1.8.19 it had some nasty problems. Use 1.8.20.

Comment (sloriot) [reply Documentation/doc/CMakeLists.txt:223]: Do you mean even the config files are broken?

File: Documentation/doc/resources/1.8.19/BaseDoxyfile.in (hunk 2 above)
Comment (albert-github) [reply Documentation/doc/resources/1.8.19/BaseDoxyfile.in:2498]: I think a good choice, also have a look at `INTERACTIVE_SVG`.

File: Documentation/doc/CMakeLists.txt (hunk 3 above)
Comment (sloriot) [reply Documentation/doc/CMakeLists.txt:223]: The cmake variable only indicates what directory to use as default for the config files, not the doxygen version to pick.

Comment (albert-github) [reply Documentation/doc/CMakeLists.txt:223]: One of the problems was with config files when they were used through a pipe (as far as I know this is not the case for CGAL).

For clarity I would not mention the number 1.8.19 here but go straight to 1.8.20 so in the future people won'yt even be tempted to use 1.8.19.

**Comment by maxGimeno** [comment]
Successfully tested in https://cgal.geometryfactory.com/CGAL/testsuite/results-5.2-Ic-37.shtml

**Pull request merged**